### PR TITLE
fix(frontend): Show correct artifact type at the side panel.

### PR DESCRIPTION
**Pull request by jlyaoyuli**
Fix the incorrect artifact type name in at the side panel.
Appended a note (for default empty file) to avoid confusing the user.

https://user-images.githubusercontent.com/56132941/181846278-7358f51a-5a29-4822-9b63-e1bcaee86211.mov

### CODE REVIEW

File: frontend/src/components/tabs/InputOutputTab.tsx [hunk 1]
```diff
@@ -62,8 +62,8 @@ export function InputOutputTab({ execution, namespace }: IOTabProps) {
   let inputArtifacts: ParamList = [];
   let outputArtifacts: ParamList = [];
   if (isSuccess && data) {
-    inputArtifacts = getArtifactParamList(filterEventWithInputArtifact(data));
-    outputArtifacts = getArtifactParamList(filterEventWithOutputArtifact(data));
+    inputArtifacts = getArtifactParamList(filterEventWithInputArtifact(data), '');
+    outputArtifacts = getArtifactParamList(filterEventWithOutputArtifact(data), '');
```
Comment (zijianjoy) [inline frontend/src/components/tabs/InputOutputTab.tsx:66]: These two lines shouldn't be using `''` for `artifactTypeName`. Can you take a video of `Execution node` and see if the output is matching what we expect? (`(This is an empty file by default)` should be applied to `'system.Metrics'` and `'system.ClassificationMetrics'` types artifacts)

Comment (jlyaoyuli) [reply frontend/src/components/tabs/InputOutputTab.tsx:66]: Video for `Execution nodes`:

https://user-images.githubusercontent.com/56132941/182493215-8b8d3a06-d0c2-4291-8d7c-bfb5ff8ff7b6.mov

File: frontend/src/components/tabs/InputOutputTab.tsx [hunk 2]
```diff
@@ -49,20 +50,44 @@ export function InputOutputTab({ execution, namespace }: IOTabProps) {
...
     { staleTime: Infinity },
   );
 
+  const { data: artifactTypes } = useQuery<ArtifactType[], Error>(
+    ['artifact_types', { linkedArtifact: linkedArtifacts }],
+    () => getArtifactTypes(),
+    {},
+  );
+
+  let artifactTypeNames: string[] = [];
```
Comment (jlyaoyuli) [inline frontend/src/components/tabs/InputOutputTab.tsx:65]: Question: we use this piece of code to get artifact type name in both `InputOutTab` and `RunDetailsV2`. Should we wrap it and put into new or existed `util` file?

https://github.com/kubeflow/pipelines/pull/8093#discussion_r938281022

**Comment by jlyaoyuli** [comment]
Show appending text in the execution nodes as well!

https://user-images.githubusercontent.com/56132941/182971632-3a60156a-6491-47dd-bbab-fcf4214270fd.mov

File: frontend/src/components/tabs/InputOutputTab.tsx (hunk 2 above)
Comment (zijianjoy) [reply frontend/src/components/tabs/InputOutputTab.tsx:65]: That is a good point, I think it makes sense to consolidate to `util` file.

File: frontend/src/mlmd/MlmdUtils.ts [hunk 3]
```diff
@@ -388,6 +388,24 @@ export function getArtifactName(linkedArtifact: LinkedArtifact): string | undefi
   return getArtifactNameFromEvent(linkedArtifact.event);
 }
 
+export function getArtifactTypeName(
```
Comment (zijianjoy) [inline frontend/src/mlmd/MlmdUtils.ts:391]: Can we add a test for this function? Thank you!

**Comment by zijianjoy** [comment]
/lgtm
/approve

Awesome work on fixing the artifact type issue!

**Pull request merged**